mlir/include/mlir/Dialect/MIGraphX/IR/MIGraphX.td

```diff
@@ -340,6 +340,17 @@ def MIGraphX_ConvolutionOp :
   }];
 }
 
+def MIGraphX_ConvolutionBwdDataOp :
+    MIGraphX_ConvOpBase<"backwards_data_convolution", [F32, F16, BF16],
+                        [F32, F16, BF16]> {
+  let summary = "Backwards data convolution";
+  let description = [{
+    The `migraphx.backwards_data_convolution` op computes gradients with
+    respect to the input tensor during back propagation, or can be used in
+    forward passes to do upsampling.
```

Comment:
nit:
It is doing convolution operation but output shape is such that it is larger than input (that is why upsampling). 
You can probably refine this description a bit.